dataland-frontend/src/utils/ValidationUtils.ts

```diff
@@ -68,3 +58,27 @@ export function isCompanyIdValid(companyId: string): boolean {
   const uuidRegexExp = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-5][0-9a-f]{3}-[089ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
   return uuidRegexExp.test(companyId);
 }
+
+export const regexSinglePage = /^[1-9]\d*$/;
+export const regexPageRange = /^[1-9]\d*-[1-9]\d*$/;
```

Comment:
Is there a reason why there are two seperates regexes and they are not combined into one?

Reply:
Doing so saved me a level of 'if' clauses, as ranges need an additional check (firstNumber < secondNumber).

Reply:
Working on it

Reply:
Done